projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadJob.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+
+export interface IDownloadJobDialogProps {
+    /**
+     * Current Path
+     */
+     parentPath: string;
+}
+
+
+export default class DownloadJobDialog extends Component<IDownloadJobDialogProps> {
+    /**
+   * Returns a React component for rendering a download menu.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+    constructor(props: IDownloadJobDialogProps) {
+        super(props);
+    }
+    /**
+    * Renders a dialog for downloading a data job.
+    *
+    * @returns React element
+    */
+    render(): React.ReactElement {
+        return (
+            <>
+                 <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobName">Job Name:</label>
+                    <input type="text" id="jobName" className='jp-vdk-input' placeholder="job-to-download" onChange={this.onNameChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobTeam">Job Team:</label>
+                    <input type="text" id="jobTeam" className='jp-vdk-input' placeholder="my-team" onChange={this.onTeamChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobRestApiUrl">Rest Api URL:</label>
+                    <input type="text" id="jobRestApiUrl" className='jp-vdk-input' placeholder="rest-api-url" onChange={this.onRestApiUrlChange} />
```

Comment:
Is there a default value here? 
I don't see where it would come from? 

I would probably set the placeholder to be 'http://my_vdk_instance' or something like that.